### DISCUSSION TITLE
Switch shadows

### DISCUSSION THREAD

**Discussion by Kris-B** [visgl/deck.gl#5651] [Q&A]
Hi everyone, I'm using a geojson layer with a sunlight shadow. How can I dynamically turn shadow off and on ? Could someone provide a code sample ? Thanks.

**Comment by Pessimistress** [comment]
Just construct a new `LightingEffect` instance and pass it to Deck's `effects` prop. See https://deck.gl/docs/developer-guide/using-lighting

**Reply by Kris-B** [reply]
Great, many thanks !
